extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -333,18 +348,54 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       dev().warn(TAG, `Frame already exists, sra: ${this.useSra}`);
       return '';
     }
+
+    const ampRtcPageElement = document.getElementById('amp-rtc');
+    let rtcRequestPromise = null;
+    let rtcConfig;
+    if (ampRtcPageElement) {
+      rtcConfig = tryParseJson(ampRtcPageElement.innerHTML);
+      if (rtcConfig && typeof rtcConfig == 'object'
```

Comment:
Is there no helper function for isObject?  Also make sure to report actionable errors via user().error